backend/benefit/applications/services/ahjo_payload.py

```diff
@@ -98,22 +99,29 @@ def _prepare_record(
     }
 
 
-def _prepare_case_records(application: Application) -> List[dict]:
-    """Prepare the list of case records"""
+def _prepare_case_records(
+    application: Application, pdf_summary: Attachment
+) -> List[dict]:
+    """Prepare the list of case records from  application's attachments,
+    including the pdf summary of the application."""
     case_records = []
     handler = application.calculation.handler
     main_document_record = _prepare_record(
         "Hakemus",
         "hakemus",
         application.created_at.isoformat(),
         application.application_number,
-        [],  # TODO Pdf version of the application goes here with prepare_record()
+        [_prepare_record_document_dict(pdf_summary)],
         handler,
     )
 
     case_records.append(main_document_record)
 
     for attachment in application.attachments.all():
```

Comment:
`attachments.exclude(attachment_type=AttachmentType.PDF_SUMMARY)` instead of `attachments.all()`

and remove `continue`